File: stack/1598_crawler_log_folder/1598_crawler_log_folder.go
package main

import "fmt"

func minOperations(logs []string) int {

	count := 0
	for i := range logs {
		if logs[i] == "../" {
			if count > 0 {
				count--
			}
		} else if logs[i] != "./" {
			count++
		}
	}

	return count
}

func main() {
	logs := []string{"d1/", "d2/", "../", "d21/", "./"}
	res := minOperations(logs)
	fmt.Println(res)
}

File: stack/1598_crawler_log_folder/1598_crawler_log_folder.py
from typing import List


class Solution:
    def minOperations(self, logs: List[str]) -> int:
        count = 0
        for log in logs:
            if log == '../':
                if count > 0:
                    count -= 1
                else:
                    pass
            elif log == './':
                pass
            else:
                count += 1
        return count


logs = ["d1/","d2/","../","d21/","./"]
res = Solution().minOperations(logs)
print(res)